docs/helm-chart/production-guide.rst

```diff
@@ -34,33 +34,54 @@ can be found at :doc:`Set up a Database Backend <apache-airflow:howto/set-up-dat
     When using the helm chart, you do not need to initialize the db with ``airflow db init``
     as outlined in :doc:`Set up a Database Backend <apache-airflow:howto/set-up-database>`.
```

Comment:
somewhere it is best to say to either use this `data.metadataSecretName`, or use 'config.database.sql_alchemy_conn', because the 1st link below refers to the 2nd link, and the 2nd link requires us to setup the config.database.sql_alchemy_conn section.

https://airflow.apache.org/docs/helm-chart/stable/production-guide.html#database
https://airflow.apache.org/docs/apache-airflow/2.4.1/howto/set-up-database.html#setting-up-a-postgresql-database

above was the reason caused me confused and i thought i had to configure both parts and still not sure which one would work;
worse yet, the page: https://airflow.apache.org/docs/apache-airflow/stable/howto/set-config.html has other values which might serve the same as "sql_alchemy_conn_" such as "sql_alchemy_conn_secret", "sql_alchemy_conn_cmd" etc

Reply:
@minnieshi -> that's why we recommend also to use `--dry-run` and `helm-diff` as tools to understand what you are doing. Airflow is highly configurable and of course - there are various ways you can configure parameters. And getting `--dry-run` and `helm-diff` as output in those cases help you to understand what you are doing. 

If you think - after understanding how it works that you could improve the docs to be less confusing, I heartily invite you to make a PR about it. 

You are probably the best person in the word to know where and how to make it less confusing (after we direct you on understanding how it works).

Airlfow has more than 2300 of contributors  who created airflow - most of them were people like you who had the problem, got a free help in better understanding how it works and gave back to the community by contributing better documentation (after they understood things). People like you - helping back in return for absolutely free software they get are the best for the community, because (like you) seem to know where and how to update documentation to be less confusing - some things are obvious for people who are longer and have some assumptions in their head that new people don't and those new people know exactly where they failed for an information when they looked for it.

This is super-easy, just click "suggest a change on this page" and you will get PR opened where you will be able to use GitHub UI to add the documentation.

Can we count on your help here, please. and opening a PR?